packages/web/src/migrations/update-11-5-2/update-root-babel-config.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { readJson, Tree } from '@nrwl/devkit';
+import { updateRootBabelConfig } from './update-root-babel-config';
+
+describe('Update root babel config', () => {
+  let tree: Tree;
+
+  beforeEach(async () => {
+    tree = createTreeWithEmptyWorkspace();
+  });
+
+  it(`should add web babel preset if it does not exist`, async () => {
```

Comment:
the name of the test probably needs to change
```suggestion
  it(`should remove web babel preset if it exists`, async () => {
```

Reply:
@jaysoo @FrozenPandaz